core/store/src/trie/mod.rs

```diff
@@ -396,7 +396,7 @@ impl RawTrieNode {
         out
     }
 
-    fn decode(bytes: &[u8]) -> Result<Self, std::io::Error> {
+    pub fn decode(bytes: &[u8]) -> Result<Self, std::io::Error> {
```

Comment:
I don’t think we need this pub.  Only pub in RawTrieNodeWithSize::decode is needed, no?